### ISSUE TITLE
Toggling Caps Lock while starting KeePassXC crashes the application

### ISSUE THREAD

**Issue by wardz** [keepassxreboot/keepassxc#5543]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )
Toggling Caps Lock while starting KeePassXC crashes the application. (Only tested on Windows 10)

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Click KeePassXC.exe
2. Quickly press Caps Lock once before the KeePassXC window is fully rendered.

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
It shouldn't crash.

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
It crashes.

## Context
[NOTE]: # ( Give us any additional information you may have. )
```
The instruction at 0x0000000000E3BE8E referenced memory at 0x0000000000000012. The memory could not be read.

Click OK to terminate the program
```

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - Version 2.6.1
Revision: 9a35bba

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows 10 Pro 64bit Build 19041 (version 2004, Norwegian Language Pack)

**Comment by droidmonkey** [comment]
I reproduced this last night as well

**Comment by droidmonkey** [comment]
Found the bug and fixed it for 2.6.2
